### ISSUE TITLE
Update Xamarin.Forms Android Project

### ISSUE THREAD

**Issue by dansiegel** [PrismLibrary/Prism.Templates#31]
## Summary

The Xamarin.Forms templates have updated the base Android project. Most notably the layout for the Tabbar and Toolbar have been removed and the styles have additionally been updated so that they will automatically come from Xamarin.Forms if not overridden by your styles.

## Intended Use Case

Simplify new Prism.Forms Android apps by reducing platform specific configuration. Note that the Styles for the Prism templates should override the styles as described in the default Xamarin.Forms Android project styles.xml so that we provide the same default Prism color scheme that we currently ship.

**Comment by Pietervdw** [comment]
After I've created a [pull request](https://github.com/PrismLibrary/Prism.Templates/pull/32), I've realized that we should probably set the Navbar colors using styles. This way the heading color will also be applied to the iOS project, e.g:

```xml
<Style TargetType ="NavigationPage">
    <Setter Property="BarBackgroundColor" Value="#1976D2"/>
    <Setter Property="BarTextColor" Value="White"/>
</Style>
```

Should we do this, or stick to using the colors specified in styles.xml?

**Comment by dansiegel** [comment]
closed by #32
